assignments/lambda-classes.js

```diff
@@ -1 +1,265 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(personalInfo) {
+        this.name =(personalInfo).name;
+        this.age =(personalInfo).age;
+        this.location =(personalInfo).location;
+        this.gender =(personalInfo).gender;
+    };
+    speak() {
+        return `Hello my name is ${this.name}, I am from ${this.location}`;
+    };
+};
+
+class Instructor extends Person {
+    constructor(instructorInfo) {
+        super(instructorInfo);
+        this.specialty = instructorInfo.specialty;
+        this.favLanguage = instructorInfo.favLanguage;
+        this.catchPhrase = instructorInfo.catchPhrase;
+    };
+    demo(subject) {
+        return `Today we are learning about ${subject}`
+    };
+    grade(student, subject) {
+        return `${student.name} receives a perfect score on ${subject}`
+    };
+};
+
+class Student extends Person {
+    constructor(studentInfo) {
+        super(studentInfo);
+        this.previousBackground = studentInfo.previousBackground;
+        this.className = studentInfo.className;
+        this.favSubjects = studentInfo.favSubjects;
+    };
+    listsSubjects() {
+        let subjects = this.favSubjects;
+        subjects.forEach(subject => console.log(subject));
```

Comment:
Good use of arrow function to keep the code terse. Could have one-lined it with `this.favSubjects.forEach`